buildbot/repos/rpm/control/centos8/Minecraft-Overviewer.spec

```diff
@@ -0,0 +1,38 @@
+Summary: Generates large resolution images of a Minecraft map.
+Name: Minecraft-Overviewer
+Version: {VERSION}
+Release: 1%{?dist}
+Source0: %{name}-%{version}.tar.gz
+License: GNU General Public License v3
+Group: Development/Libraries
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-buildroot
+Vendor: Andrew Brown <brownan@gmail.com>
+Url: http://overviewer.org/
+Requires: epel-release python3-pillow python3-numpy
+BuildRequires: epel-release git python36-devel python3-numpy
+
+%description
+The Minecraft Overviewer is a command-line tool for rendering high-resolution
+maps of Minecraft worlds. It generates a set of static html and image files and
+uses the Google Maps API to display a nice interactive map.
+
+%prep
+git clone https://github.com/python-pillow/Pillow.git %{_tmppath}/Pillow
```

Comment:
Instead of cloning whatever is the current git master, consider checking out the precise version that's used in CentOS 8, which right now is 5.1.x:
```
wget -P %{_tmppath} https://github.com/python-pillow/Pillow/archive/5.1.0.tar.gz
tar -xf %{_tmppath}/5.1.0.tar.gz --directory %{_tmppath}
```

`PIL_INCLUDE_DIR` should then be `%{_tmppath}/Pillow-5.1.0/src/libImaging`

Reply:
done